src/Item/PluginItem/Interpretations/Interpretation.js

```diff
@@ -270,10 +277,14 @@ class Interpretation extends Component {
                     </span>
                 </div>
                 {this.renderCommentOrEditField(comment)}
-                {this.userIsOwner(comment.user.id) &&
-                    editButton(() => this.toggleEdit(comment.id))}
+                {this.userIsOwner(comment.user.id) && (
+                    <EditButton
+                        action={() => this.toggleEdit(comment.id)}
+                        text={this.getEditText(comment.id)}
+                    />
+                )}
                 {this.hasDeleteAccess(comment.user.id) &&
-                    deleteButton(() => this.deleteComment(comment.id))}
+                    DeleteButton(() => this.deleteComment(comment.id))}
```

Comment:
shouldn't this be `DeleteButton({action: () => this.deleteComment(comment.id)})`

Reply:
It should actually be <DeleteButton action={........................